### DISCUSSION TITLE
safari浏览器无法播放视频和下载文件

### DISCUSSION THREAD

**Discussion by LovelyTitan** [AlistGo/alist#2065] [General]
allist版本是3.2.0
使用mac的safari浏览器或是iphone的safari浏览器，测试多个视频文件（含本地文件、网盘文件）基本上都无法正常打开或是下载。然而同样的文件使用windows的edge浏览器（chrome内核）却是完全正常的。

**Comment by zjb19981222** [comment]
浏览器解码问题

**Reply by CyberXfuck** [reply]
应该不是解码的问题，我的Safari浏览器曾经成功的播放了MKV。然而现在连MP4都播放不了了。。。感觉像是哪里设置的问题。。。

**Reply by CyberXfuck** [reply]
![image](https://github.com/alist-org/alist/assets/118439882/367f0781-91ec-4ab1-bafb-96f13a90d8e6)
不知道抽什么风，Safari又可以正常的播放MKV了。。。

**Comment by iemobiu** [comment]
我也遇到了同样的问题。如果是浏览器解码问题，那么这个兼容性问题感觉做的比较一般。测试了很多视频，感觉只有很老的MP4格式手机（iPhone）上的浏览器才可以播放。不知道能否解决下这个问题呢？
